save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CollectionView.kt

```diff
@@ -52,6 +47,13 @@ class CollectionView : AbstractView<PropsWithChildren, State>(false) {
                             a(href = "#/${it.value.owner}/${it.value.name}") {
                                 +it.value.name
                             }
+                            span("border ml-2 pr-1 pl-1 text-xs text-muted ") {
```

Comment:
Should we extract it into a separate function?

Reply:
yes, I was making it right now :)

Reply:
fixed